esx_service/vmdk_ops.py

```diff
@@ -648,7 +653,9 @@ def disk_attach(vmdk_path, vm):
                      if type(dev) == vim.VirtualDisk and dev.controllerKey ==
                      controller_key])
         # search in 15 slots, with unit_number 7 reserved for scsi controller
-        availSlots = set(range(0, 6) + range(8, 16)) - taken
+        range1 = set(range(0, 7))
```

Comment:
Missed the "saving lines" section :-)), anyway I got the same approach
above.

On Tue, Jun 28, 2016 at 11:07 AM, Mark Sterin <notifications@github.com>
wrote:

> In esx_service/vmdk_ops.py
> <https://github.com/vmware/docker-volume-vsphere/pull/479#discussion_r68699882>
> :
>
> > @@ -648,7 +653,9 @@ def disk_attach(vmdk_path, vm):
> >                       if type(dev) == vim.VirtualDisk and dev.controllerKey ==
> >                       controller_key])
> >          # search in 15 slots, with unit_number 7 reserved for scsi controller
> > -        availSlots = set(range(0, 6) + range(8, 16)) - taken
> > +        range1 = set(range(0, 7))
>
> @govint <https://github.com/govint> - reading manual helps.
>
> availSlots = ( set(range(0,7))  | set(range(7, 15)) )  - taken
>
> works fine on both pyth2 and pyth3
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/479/files/618f13068273be9c4251cba120e77e704c061ed4#r68699882>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseBUtFrYZtbaucwWjI2ZSWEzIdspEks5qQLMqgaJpZM4I4qrt>
> .
>